src/json-patch-ot-agent.js

```diff
@@ -73,3 +73,14 @@ JSONPatchOTAgent.applyOT = function(callback){
         callback(obj, consecutivePatch);
 	};
 };
+
+/**
+ * Reset queue internals and object to new, given state
+ * @param obj object to apply new state to
+ * @param newState versioned object representing desired state along with versions
+ */
+JSONPatchOTAgent.prototype.reset = function(obj, newState){
+	this.ackLocalVersion = newState[this.localPath.replace(/^\//, '')];
```

Comment:
I think it's highly likely, even we have considered it already.

I believe if we would like to make people adopt this protocol we have to make it clean and bug-less. 
To prove the theory works before proposing it as a standard.